### ISSUE TITLE
Financial support to the project

### ISSUE THREAD

**Issue by arnaudgo** [gcp/leela-zero#562]
Hi!

It would be nice to provide ability to support financially the project :-)

So new computing resources could be purchased or beer could be paid to all the contributors... Whatever use, do you think it's a good idea?

Arnaud

**Comment by Yakago** [comment]
I'm not sure beer would propel the project forward :-) 

However, I believe it is rather complicated to do financial support apart from using cloud services, that are unfortunately quite cost inefficient.

For long term support, it seems that buying actual hardware is more efficient, and that kind of support seems difficult to manage.

But I'm sure many would agree that is a good idea if a proper model can be found.

**Comment by jjoshua2** [comment]
There will be the successor to the 1080 ti in a couple months, and if a lot of people chipped in a few bucks we could save up by then and get it for gcp. We could use gofundme or just paypal... He already uses a 1080 ti for training, but a faster one would make the new models to test come out faster.

**Comment by odeint** [comment]
AFAIK it will be more than just a few months until GA104, and GA102 may even slip into 2019. More Volta products (after the Titan V) aren't expected either, at least in the gamer market.

**Comment by jjoshua2** [comment]
Well if the fund gets up to a titan v before a cheaper one comes that's
just as well.

**Comment by prusswan** [comment]
Does not have to be a Titan, just get the cards with the best price to performance ratio

**Comment by fell111** [comment]
Titan V is useful for training at FP16 and FP64. If we insist on FP32, 1080 ti is enough.

**Comment by gcp** [comment]
I don't need and don't want money. I tested a V100 and it's not useful because fp16 runs into accuracy issues. If training would be a bottleneck then I'll just get a second 1080 Ti or rent one on AWS and hope someone better with TensorFlow fixes #35.

**Comment by jkiliani** [comment]
Would this still apply even for a 20x256 (or larger) network? How many training steps per hour can you run on your machine then, are you optimistic that it still won't be a problem?

**Comment by gcp** [comment]
About 10 times less, which is about the slowdown for the amount of games coming in because every client gets a lot slower. So the speed really remains the same. It would be more of an issue if the amount of clients would get 10 times bigger.
